src/view/QuestionsList.js

```diff
@@ -0,0 +1,93 @@
+//da-l in ma0sa, il harcodez
+
+import 'bulma/css/bulma.css';
+
+
+
+
+const QuestionsList = ({ questions, onCreateQuestion, searchTitle, searchTag, onChange }) => (
+
+            <div className={"column"}>
+                    <section className="hero is-fullheight is-dark is-bold">
+                            <div className="hero-body">
+                                    <div className="container">
+                                            <h1 className="title">
+                                                    Numberless questions
+                                            </h1>
+                                            <h2 className="subtitle">
+                                                    Reveal any mistery...scroll into the abyss
+                                            </h2>
+                                    </div>
+                            </div>
+                    </section>
+                    <section className="hero is-light is-bold">
+                            <div className={"columns"}>
+                                    <div className={"column"}>
+                                    </div>
+                                    <div className={"column"}>
+                                            <div className={"notification"}>
+                                                    <label> Search question by title or tag </label>
+                                                    <input  className={"input"} onChange={e => onChange("toSearch", e.target.value)} />
+                                                    <div className={"columns"}>
+                                                            <div className={"column"}>
+                                                                    <button className={"button is-dark"} onClick={searchTitle}>Search by title</button>
+                                                            </div>
+                                                            <div className={"column"}>
+                                                                    <button className={"button is-dark"} onClick={searchTag}>Search by tag</button>
+                                                            </div>
+
+                                                    </div>
+
+
+                                            </div>
+                                            <button className={"button is-dark"} onClick={onCreateQuestion}>Add new question</button>
+                                    </div>
+                                    <div className={"column"}>
+                                    </div>
+
+                            </div>
+                            <br/>
+                            <br/>
+
+
+                            {
+                                    questions.map((question, index) => (
+                                        <div className={"columns"}>
```

Comment:
When doing map operations in JSX, assign a key (with the index) to the top-most JSX element (in this case, to the "columns" div). If you check the developer tools console, you are most likely getting an error / warning from React about this.